.gitignore

```diff
@@ -6,3 +6,22 @@
 /ssh.*
 .env
 codeclimate.json
+
+# states for tests
+/tests/Functional/application-state-new-files-only/expected/data/out/state.json
```

Comment:
bych to klidně šoupnul do .gitignore přímo do `/tests/Functional/application-state-new-files-only/expected/data/out/`?

Reply:
to je dobrý nápad

Reply:
Přímo do složky data to nejde, porovnávají se všechny soubory, zatím jsem to dal do složky konkrétního testu, viz https://github.com/keboola/aws-s3-extractor/pull/50/commits/bc98cb8f51d555a2823d097dbc3e47c65d4662fe